pkgs/by-name/za/zatackax/package.nix

```
@@ -0,0 +1,44 @@
+{
+  autoreconfHook,
+  fetchFromGitHub,
+  SDL,
+  SDL_image,
+  SDL_mixer,
+  SDL_ttf,
+  stdenv,
+  lib,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zatackax";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "simenheg";
+    repo = "zatackax";
+    rev = "v${version}";
+    hash = "sha256-1m99hi0kjpj5Yl1nAmwSMMdQWcP0rfLLPFJPkU4oVbM=";
+  };
+
+  nativeBuildInputs = [
+    SDL_mixer
+    SDL_ttf
+    SDL_image
+    SDL
+    autoreconfHook
+  ];
```

```suggestion
  strictDeps = true;

  nativeBuildInputs = [
    autoreconfHook
  ];

  buildInputs = [
    SDL_mixer
    SDL_ttf
    SDL_image
    SDL
  ];
```
> buildInputs: These are often programs and libraries used by the new derivation at run-time
https://nixos.org/manual/nixpkgs/unstable/#var-stdenv-buildInputs

> nativeBuildInputs: These are programs and libraries used at build-time
https://nixos.org/manual/nixpkgs/unstable/#var-stdenv-nativeBuildInputs

> strictDeps: By default, Nix doesn’t check that the dependencies do respect this difference, unless you use `strictDeps = true.`